pkg/agent/route/route_linux_test.go

```diff
@@ -1242,94 +1246,6 @@ func TestDeleteNodePort(t *testing.T) {
 	}
 }
 
-func TestAddClusterIPRoute(t *testing.T) {
```

Comment:
Should we still test serviceCIDRHandler?

Reply:
Added.